q2studio/api/workspace.py

```diff
@@ -99,7 +104,9 @@ def inspect_artifact(uuid):
     except Exception:
         abort(404)
 
-    return jsonify({'uuid': metadata.uuid, 'type': metadata.type})
+    return jsonify({'uuid': metadata.uuid,
+                    'type': metadata.type,
+                    'source_format': metadata.source_format})
```

Comment:
Should be `metadata.format`, but again there's nothing on the client-side to draw this information, so it doesn't matter much. I would remove.